agent_deploy/IBMCloud-Kubernetes-Service/install-agent-k8s.sh

```diff
@@ -109,6 +109,24 @@ function install_k8s_agent {
 
     CONFIG_FILE=/tmp/sysdig-agent-configmap.yaml
 
+    echo "* Retreiving the IKS Cluster ID and Cluster Name"
+    IKS_CLUSTER_ID=$(kubectl get cm -n kube-system cluster-info -o yaml | grep ' "cluster_id": ' | cut -d'"' -f4)
+    CLUSTER_NAME=$(ic ks clusters -s | grep $IKS_CLUSTER_ID | cut -d' ' -f1)
```

Comment:
My only concern with this is the following:
- it is possible to be using a cluster belonging to IBM Cloud account A while being logged in at CLI level to customer B
- this is obviously not ideal but technically possible
- I believe it's also possible that the CLI credentials have expired and that the command could fail
- as a minimum I'd like to see written somewhere that a prerequisite to a correct outcome from this script is to be logged in to the correct IBM cloud account at CLI level
- another option could be to use IBM Cloud CLI API keys passed to the script as an argument to perform a non-interactive login to the CLI: 
```
ibmcloud login -a https://api.ng.bluemix.net --apikey @/Users/neo/secrets/benedetto-ibm-account-apikeys.json
```